server/main.py

```diff
@@ -45,11 +48,6 @@ async def room(websocket: WebSocket) -> None:
     try:
         while True:
             event = await client.receive()
-            buggy, sender, event_data = await handler(event, room_code)
-
-            await manager.broadcast(event_data, room_code, sender=sender, buggy=buggy)
-    except WebSocketDisconnect as err:
-        await manager.broadcast(err.response, room_code, sender=client)
-        manager.disconnect(client, room_code)
-    except NotImplementedError as err:
-        await client.send(err.response)
```

Comment:
If we handle the errors on the EventHandler class, we no longer need to try/except this